src/components/Radio/RadioItem/RadioItem.module.css

```diff
@@ -0,0 +1,102 @@
+.radioInput {
+  height: 0;
+  width: 0;
+  opacity: 0;
+  position: absolute;
+}
+
+.radioInput + .radioButtonContainer {
+  cursor: pointer;
+  display: flex;
+  border-radius: var(--border-radius-small);
+  padding: 10px 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  flex: 1 1 0;
+  width: auto;
+}
+
+.radioInput + .radioButtonContainer .header {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.radioInput + .radioButtonContainer .header .checkBox {
+  width: 16px;
+  height: 16px;
+  border-radius: 50%;
+  border: solid 2px var(--font-color-sub);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-right: 14px;
+}
+
+.radioInput + .radioButtonContainer .header .checkBox .checkBoxFilling {
+  width: 0px;
+  height: 0px;
+  border-radius: 50%;
+  background-color: var(--font-color-sub);
+}
+
+.radioInput + .radioButtonContainer .header .title {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.radioInput:disabled + .radioButtonContainer .header .title {
+  color: var(--color-disabled-font);
+}
+
+.radioInput + .radioButtonContainer .description {
+  color: var(--font-color-sub);
+  font-size: 13px;
+  margin: 0px 0px 0px 30px;
+}
+
+.radioButtonContainer:hover {
+  background-color: var(--color-primary);
```

Comment:
If the primary color is blue, the font color should also change to white or something.
Maybe add `color: var(--color-font-primary-font)`

To prevent this in the future we could create a theme selector in the docs to preview the component for another theme. Should I add an issue for this?

Reply:
Yes! Good idea!

Reply:
This is a hard one however. Because when not selected the background is white. Meaning white text will not work

Reply:
See #36